module/Olcs/config/module.config.php

```diff
@@ -67,6 +67,11 @@
 use Olcs\Controller\Lva\Factory\Controller\Variation as LvaVariationControllerFactories;
 use Olcs\Controller\Lva\Licence as LvaLicenceControllers;
 use Olcs\Controller\Lva\Variation as LvaVariationControllers;
+use Olcs\Controller\Messages\ApplicationConversationListController;
```

Comment:
Not on you, but at somepoint we should refactor these to
`Olcs\Controller\Messaging\*`
to match Commandhandler etc